Makefile

```diff
@@ -1,11 +1,13 @@
 CRYSTAL_PATH ?= $(shell pwd)/share/crystal-ic/src
 CRYSTAL_CONFIG_PATH ?= '$$ORIGIN/../share/crystal-ic/src'
+CRYSTAL_LIB_CONFIG_PATH ?= '$$ORIGIN/../lib/ic/share/crystal-ic/src'
```

Comment:
For shard installation, point to the shard's shared standard library instead of assuming it's installed alongside the `bin` directory